### PR TITLE
Yahoo 🗻

### PR DESCRIPTION
**Pull request by jsom**

### CODE REVIEW

File: TradeItIosTicketSDK2/CloseableViewController.swift [hunk 1]
```diff
@@ -16,9 +16,9 @@ class CloseableViewController: UIViewController {
             self.createCloseButton()
         }
 
-        TradeItThemeConfigurator.configureBarButtonItem(button: self.navigationItem.leftBarButtonItem)
-        TradeItThemeConfigurator.configureBarButtonItem(button: self.navigationItem.rightBarButtonItem)
-        TradeItThemeConfigurator.configureBarButtonItem(button: self.navigationItem.backBarButtonItem)
+        //TradeItThemeConfigurator.configureBarButtonItem(button: self.navigationItem.leftBarButtonItem)
+        //TradeItThemeConfigurator.configureBarButtonItem(button: self.navigationItem.rightBarButtonItem)
+        //TradeItThemeConfigurator.configureBarButtonItem(button: self.navigationItem.backBarButtonItem)
```
Comment (jsom) [inline TradeItIosTicketSDK2/CloseableViewController.swift]: These need to be moved. Yahoo uses `CloseableViewController` but not the themes.

File: TradeItIosTicketSDK2/TradeItSelectionViewController.swift [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import UIKit
 
-class TradeItSelectionViewController: TradeItViewController {
+class TradeItSelectionViewController: TradeItYahooViewController {
```
Comment (guillaumedebavelaere) [inline TradeItIosTicketSDK2/TradeItSelectionViewController.swift]: Is the TradeItSelectionViewController used by Yahoo ?

Comment (jsom) [inline TradeItIosTicketSDK2/TradeItSelectionViewController.swift]: Ah good point. I think it's used in both SDKs. Might need to split this out.

File: TradeItIosTicketSDK2/TradeItYahooNavigationController.swift [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+        let gradientView = YFTopGradientView(frame: self.view.frame)
+        gradientView.translatesAutoresizingMaskIntoConstraints = false
+        self.view.addSubview(gradientView)
+        self.view.sendSubview(toBack: gradientView)
+        self.view.addConstraints([
+            gradientView.topAnchor.constraint(equalTo: self.view.topAnchor),
+            gradientView.widthAnchor.constraint(equalTo: self.view.widthAnchor),
+            gradientView.centerXAnchor.constraint(equalTo: self.view.centerXAnchor),
+            gradientView.heightAnchor.constraint(equalToConstant: 64.0)
```
Comment (guillaumedebavelaere) [inline TradeItIosTicketSDK2/TradeItYahooNavigationController.swift]: Is 64 a constant for all screens ?

**Pull request merged**